Source/UnitTests/GTMSessionFetcherFetchingTest.m

```diff
@@ -1849,6 +1852,102 @@ - (void)testInsecureRequests_WithoutFetcherService {
   [self testInsecureRequests];
 }
 
+- (void)testCollectingMetrics_WithSuccessfulFetch API_AVAILABLE(ios(10.0), macosx(10.12),
```

Comment:
We generally have versions of all the tests both with and without a fetcher service (_WithoutFetcherService), so you should also have versions of these without (the default is with).

Reply:
Done.